lib/Sema/CSSimplify.cpp

```diff
@@ -4774,6 +4834,46 @@ ConstraintSystem::SolutionKind ConstraintSystem::simplifyMemberConstraint(
                                       flags, locatorB);
     };
 
+    // Check if any property wrappers on the base of the member lookup have
+    // mactching members that we can fall back to.
+    if (auto dotExpr =
+            dyn_cast_or_null<UnresolvedDotExpr>(locator->getAnchor())) {
+      auto baseExpr = dotExpr->getBase();
+      auto resolvedOverload = getResolvedOverloadSets();
+      while (resolvedOverload) {
+        if (resolvedOverload->Locator->getAnchor() == baseExpr)
+          break;
+        resolvedOverload = resolvedOverload->Previous;
+      }
+
+      if (resolvedOverload && resolvedOverload->Choice.isDecl()) {
```

Comment:
See my previous comment, if the logic related to finding wrapper is extracted into its own function it could be reused here.